lib/odoo.js

```diff
@@ -38,11 +38,18 @@ class Odoo {
 		};
 
 		return got.post(`${config.host}:${config.port}/web/session/authenticate`, {json: true, headers, body})
-			.then(res => new OdooClient(res.body.result, {
-				host: config.host,
-				port: config.port,
-				sid: res.headers['set-cookie'][0].split(';')[0]
-			}));
+			.then(res => {
+				if (res.body.error) {
+					// HTTP Status is 200 / OK!
+					throw res.body.error.message;
```

Comment:
I'm saying that you should throw `res.body.error`, not `res.body.error.message`. Thus, `res.body.error` is an object

```js
{
    message: 'Odoo Server Error',
    code: 200,
    data: {
        debug: 'Traceback ...'
    }
}
```